src/test/java/org/sonarsource/sonarlint/ls/mediumtests/ConnectedModeMediumTests.java

```diff
@@ -240,7 +240,7 @@ void analysisConnected_matching_server_issues_on_sq_with_pull_issues() {
     mockWebServerExtension.addStringResponse("/api/system/status", "{\"status\": \"UP\", \"version\": \"9.6\", \"id\": \"xzy\"}");
 
     mockWebServerExtension.addProtobufResponseDelimited(
-      "/api/issues/pull?projectKey=myProject&branchName=master&languages=apex,c,cpp,css,web,java,js,php,plsql,py,secrets,ts,xml,yaml,go",
+      "/api/issues/pull?projectKey=myProject&branchName=master&languages=apex,c,cpp,css,web,java,js,php,plsql,py,secrets,ts,xml,yaml,go,cloudformation,docker,kubernetes,terraform",
```

Comment:
Nitpick: maybe we should extract this concatenated list of languages in a constant in this test, to avoid "shotgun surgery" each time we add a new language.